server/public/script/script.js

```diff
@@ -0,0 +1,9 @@
+let articleListContent = document.getElementsByClassName('articleList-content');
+
+    for (var i = 0; i < articleListContent.length; i++) {
+
+      if (articleListContent[i].innerHTML.length > 70) {
```

Comment:
you dont need the `if` check, you can just slice and if it's less then nothing will happen.

Reply:
@kabaros I update it